server/src/main/java/org/apache/iotdb/db/qp/physical/sys/ClearCachePlan.java

```diff
@@ -34,4 +34,9 @@ public ClearCachePlan() {
   public List<Path> getPaths() {
     return new ArrayList<>();
   }
+
+  @Override
+  public List<String> getPathsStrings() {
+    return new ArrayList<>();
```

Comment:
Use Collections.emptyList().

Reply:
I'll change it. Thanks for your suggestion.